src/main/java/com/chromediopside/service/ProfileService.java

```diff
@@ -0,0 +1,57 @@
+package com.chromediopside.service;
+
+import com.chromediopside.model.GiTinderProfile;
+import com.chromediopside.model.Language;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.eclipse.egit.github.core.Repository;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.service.RepositoryService;
+import org.eclipse.egit.github.core.service.UserService;
+
+public class ProfileService {
+
+  private static final String GET_REQUEST_IOERROR = "Some GitHub data not available for this accessToken!";
+
+  public GiTinderProfile getProfileFromGitHub(String accessToken) {
+    GiTinderProfile giTinderProfile = new GiTinderProfile();
```

Comment:
Use dependency injection whenever you can. There are some instantiations here where you could use dependency injection as well.